tests/test_quoteengine.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+import pytest
+
+from tululbot.utils import QuoteEngine
+
+
+class FakeResponse:
+    def get_json(self):
+        return self.json
+
+    pass
+
+
+@pytest.fixture
+def qe():

```

Comment:
before that, why did you put this as fixture? as I understand it, this is your system under test. shouldn't you explicitly instantiate `QuoteEngine` inside your test case and not provide this as fixture? this is not going to be used anywhere else (at least for now).